tests/forms_tests/tests/test_formsets.py

```diff
@@ -617,6 +620,32 @@ def test_formsets_with_ordering(self):
             {'votes': 900, 'ORDER': 2, 'choice': 'Fergie'},
         ])
 
+    def test_formsets_with_order_custom_widget(self):
+        class FormSet(BaseFormSet):
+            ordering_widget = HiddenInput
+
+        class ArticleForm(ModelForm):
+            class Meta:
+                model = Article
+                fields = ('content', )
+
+        ChoiceFormSet = formset_factory(ArticleForm, formset=FormSet, can_order=True)
```

Comment:
`ChoiceFormSet` -> `ArticleFormSet`

You mixed `Article` with `Choice` in few places.